.github/workflows/macos.yaml

```diff
@@ -31,6 +31,8 @@ on:
 jobs:
   ci:
     runs-on: macos-13
+    env:
+      CMAKE_OSX_DEPLOYMENT_TARGET: "13.0"
```

Comment:
See https://github.com/dhermes/bezier/pull/318/files#r1643786071 for earlier context on this (this is meant to be temporary!)